src/store/actionCreators/virtualStudies.js

```diff
@@ -0,0 +1,59 @@
+import {
+  LOAD_VIRTUAL_STUDY,
+  SET_ACTIVE_INDEX,
+  SET_SQONS,
+  SET_VIRTUAL_STUDY_ID,
+} from '../actionTypes';
+import { getVirtualStudy } from 'services/virtualStudies';
+import { initializeApi } from 'services/api';
+
+const api = initializeApi({
+  onError: console.err,
+  onUnauthorized: response => {
+    console.warn('Unauthorized', response);
+  },
+});
+
+export const loadSavedVirtualStudy = virtualStudyId => {
+  return dispatch =>
+    getVirtualStudy(api)(virtualStudyId)
+      .then(virtualStudy => {
+        const {
+          uid,
+          content: { sqons, activeIndex },
+        } = virtualStudy;
+
+        dispatch({
+          type: LOAD_VIRTUAL_STUDY,
```

Comment:
@jberube For good practices, in general I would use the pattern:

- VIRTUAL_STUDY_REQUESTED
- VIRTUAL_STUDY_SUCCESS
- VIRTUAL_STUDY_FAILURE

The main reasons behind this pattern are:

- More leeway in terms of different states in regards to a specific async action
- Easier to understand the flow of the application through the debug tools
- Re-usable across all async actions

Reply:
I agree. I tend to wait until I need at least two of those actions to name them this way, but I don't care about doing that up-front as a general practice for asynchronous actions.

Reply:
Might as well start this one on the right foot! 👍